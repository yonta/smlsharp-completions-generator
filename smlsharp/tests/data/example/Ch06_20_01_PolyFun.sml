op ::;
fun cons e L = e :: L;
fun twice f x = f (f x);
