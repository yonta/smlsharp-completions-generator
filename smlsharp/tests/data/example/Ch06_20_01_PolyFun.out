val it = fn : ['a. 'a * 'a list -> 'a list]

val cons = fn : ['a. 'a -> 'a list -> 'a list]

val twice = fn : ['a. ('a -> 'a) -> 'a -> 'a]
